### PR TITLE
Feature/122 command sending

### PR DESCRIPTION
**Pull request by No311**
Perhaps in progress, tell me what you want changed if so.

Changes:
-The introduction of sendToServer for all our HTTP PUT needs,
-The implementation of sendToServer into all components that need it (BottomBar, RefboxField, RefboxSettings, Settings),
-The creation of tests for the new function,
-The removal of some obsolete props in RefboxSettings (as well as the proptypes), which followed from the above three changes.

### CODE REVIEW

File: front/src/RefboxField/RefboxButton.js [hunk 1]
```diff
@@ -17,9 +17,7 @@ const RefboxButton = props => {
     <Button
       className={props.className}
       teamColor={props.teamColor}
-      onClick={() => {
-        props.onClick();
-      }}
+      onClick={() => props.onClick()}
```
Comment (helaan) [inline front/src/RefboxField/RefboxButton.js]: out of scope of this pr

File: front/src/RefboxSettings/index.js [hunk 2]
```diff
@@ -18,18 +18,18 @@ import PropTypes from "prop-types";
  * - onClickDropBall: a function on what to do when DropBall is pressed
  * - onClickInOut: a function on what to do when InOutButton is pressed
```
Comment (helaan) [inline front/src/RefboxSettings/index.js]: If you remove props, remove them from the comments as well

File: front/src/App.js [hunk 3]
```diff
@@ -162,12 +162,8 @@ class App extends Component {
         <Container>
           {activePage === "refbox" && (
             <div>
-              <RefboxField onClick={(tag, teamColor) => {}} />
-              <RefboxSettings
-                onClickDropBall={() => {}}
-                onClickInOut={prop => {}}
-                onChange={(type, value) => {}}
-              />
+              <RefboxField isPenalty={false} />
```
Comment (helaan) [inline front/src/App.js]: Does this just disable penalty move completely?

Comment (No311) [reply front/src/App.js]: Not quite sure. We should have a branch for completely using the penalty box or something. It still exists, at the very least.

**Pull request merged**